components/brave_rewards/browser/net/network_delegate_helper.cc

```diff
@@ -67,7 +67,7 @@ void GetRenderFrameInfo(const net::URLRequest* request,
   }
   if (!content::ResourceRequestInfo::GetRenderFrameForRequest(
           request, render_process_id, render_frame_id)) {
-    const content::WebSocketHandshakeRequestInfo* websocket_info =
+    content::WebSocketHandshakeRequestInfo* websocket_info =
```

Comment:
should this commit be squashed together with https://github.com/brave/brave-core/pull/2829/commits/ad295392b2a2ee5f5ba6172d20fb8f95c45451ba ? (since they address the same Chromium commit)

Reply:
Yep!